addons/portal/views/portal_templates.xml

```diff
@@ -6,31 +6,25 @@
             <attribute name="t-attf-class" add="#{'o_portal' if is_portal else ''}" separator=" "/>
         </xpath>
         <xpath expr="//div[@id='wrapwrap']/header/img" position="replace">
-            <nav class="navbar navbar-expand-md navbar-light bg-light">
-                <div id="top_menu_container" class="container">
+            <nav class="navbar navbar-expand navbar-light bg-light">
```

Comment:
If you just install Portal, I don't think there is a need for navbar-collapse, as now, the only component is the user dropdown. So in a mobile, a navbar-collapse for just this component would be counter-productive. I think the opposite (navbar-expand = never collapse) would be a good idea. But maybe, I'm missing something.

Reply:
navbar-expand = never collapse indeed but you removed the collapse so the class is useless, it will already never collapse... or I am missing something. But ok on the principle

Reply:
It's mostly about the CSS rules that comes with `.navbar-expand`, like padding.